app/src/main/java/org/wikipedia/readinglist/database/ReadingListDbHelper.java

```diff
@@ -682,7 +748,7 @@ public ReadingListPage getPageById(long id) {
     }
 
     @Nullable
-    private ReadingListPage getPageByTitle(SQLiteDatabase db, @NonNull ReadingList list, @NonNull PageTitle title) {
```

Comment:
What's the rationale for changing this to use the `readingListId`?  It would be much cleaner to use structured objects where possible.

Reply:
Make sense. Done!